src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor());
```

Comment:
Let's initialize the ColorSupplier as I suggested above, also passes the number to the Ball constructor ;)